pkg/operator/ceph/config/config.go

```diff
@@ -113,14 +113,32 @@ func SetDefaultConfigs(
 		return errors.Wrapf(err, "failed to apply default Ceph configurations")
 	}
 
+	// When enabled the collector will logrotate logs from files
+	if clusterSpec.LogCollector.Enabled {
+		// Override "log file" for existing clusters since it is empty
+		logOptions := []Option{
+			configOverride("global", "log file", "/var/log/ceph/$cluster-$name.log"),
```

Comment:
If all the daemons are logging to `/var/log/ceph`, if there are multiple daemons on the same node, would logrotate conflict (or at least be redundant) for the daemons?

Reply:
No, it's handled, we only rotate each daemon's log individually.